### DISCUSSION TITLE
- Reasons for Migration in Bangalore (2011)

### DISCUSSION THREAD

**Discussion by giscus** [DACSS/DACSS_601_August2022_v2#260] [Announcements]
# - Reasons for Migration in Bangalore (2011)

Final Project

https://dacss.github.io/DACSS_601_August2022_v2/posts/Mekhala%20Final.html

**Comment by LarriMill** [comment]
Hi Mekhala, WOW! This is so thorough, I am very impressed. Absolutely lovely job of describing your reasoning, the context of your data, and what your audience needs to know. I hope you continue this work because this is great. My suggestion is to make sure that your graph colors are easily interpreted-- for example, the "reasons split by last place of residence" graph was a bit too dark to interpret.

**Reply by mekh09** [reply]
Thank you!
